x-pack/plugins/index_lifecycle_management/public/extend_index_management/components/index_lifecycle_summary.tsx

```diff
@@ -82,34 +82,20 @@ interface Props {
   index: Index;
   getUrlForApp: ApplicationStart['getUrlForApp'];
 }
-interface State {
-  showStackPopover: boolean;
-  showPhaseExecutionPopover: boolean;
-}
 
-export class IndexLifecycleSummary extends Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      showStackPopover: false,
-      showPhaseExecutionPopover: false,
-    };
-  }
-  toggleStackPopover = () => {
```

Comment:
stack related functions were unused and seems to be leftover code from https://github.com/elastic/kibana/pull/128397